web/assets/full-width-container.js

```diff
@@ -3,15 +3,16 @@ const fullWidthContainer = () => {
     const headerContainer = document.getElementsByClassName(
       "moj-header__container",
     )[0];
-    const mainContainer = document.getElementsByClassName(
-      "govuk-width-container",
-    )[0];
-    if (headerContainer && mainContainer) {
+    const mainContainers = document.querySelectorAll(
+      ".govuk-width-container",
+    );
```

Comment:
This change is to make everything full width. The original code which targeted the first "govuk-width-container" element was now the phase banner under the main header. So this change is to expand all containers to full width